actors/evm/src/lib.rs

```diff
@@ -51,12 +51,16 @@ fn test_method_selector() {
 #[repr(u64)]
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
-    // TODO: Do we want to use ExportedNums for all of these, per FRC-42?
     InvokeContract = 2,
     GetBytecode = 3,
     GetStorageAt = 4,
     InvokeContractDelegate = 5,
     GetBytecodeHash = 6,
+    // FRC-42 method exports
+    InvokeContractExported = frc42_dispatch::method_hash!("InvokeContract"),
```

Comment:
This should be something like `InvokeEVMContract`.

Reply:
Is this a renaming of `InvokeContract` as well?

Reply:
i.e. `InvokeEVMContractExported = frc42_dispatch::method_hash!("InvokeEVMContract"),`

Reply:
Just the method hash. There is no implicit namespacing, so we need to specify _what_ we're talking about here.